coursebook/week-5/project.md

```diff
@@ -1,23 +1,55 @@
+The key difference between this project and your API week project is that you will be making your **API calls from the back-end** and **testing your server**.
 
-### Goals
 
-The primary aim of this project is to get comfortable with setting up an Express server and to discover the awesomeness of Handlebars! Spend time considering how to format your layouts, partials and helper functions.
+### Example:
 
-Previous cohorts have built apps where users can share, for example, blog posts, jokes, photos etc. But the app can be about anything as long as you are covering project's technology tasks and goals.
+#### User Stories:
+'As a student at F&C I want to know all the train departure times from Finsbury Park tube station, so that I can get home in time for dinner'.🚉🍛
 
-### What's also important:
+'As a regular commuter, I want to input which direction of travel I am interested in so that I can see information that is relevant to me.'
 
-- Include tests and set up code coverage.
-- We expect to see unit tests of pure functions, and integration tests of your server routes, including tests for errors such as 404.
-- You should write tests for your database and use a separate test database and build script.
-- Use Heroku or a similar service to host the app and the database.
-- Try to include ES6 syntax on the server.
+This news feed could be created with data provided via the [TFL API](https://api.tfl.gov.uk/).
+
+You can build on this user story or create your own user stories as long as they are consistent and the below specs are fulfilled.
+
+### Goals:
+1) Write your server using expressJS
+
+2) Use at least 1 API that uses an Access Key and make your API calls from the back-end (try using the Request node module)
+
+3) Add the api data to your webpage using DOM manipulation on the Front End
+
+3) Your server should contain a minimum of 2 routes
+
+4) We expect to see lots of tests! Modularise your code and test all your pure functions. Write tests for as much of your back-end and front-end logic as you can. We don't expect tests on the DOM.
+
+5) Test your server routes by injecting fake HTTP requests using Supertest (including testing for 404's). _Note - you are not required to test any server route that makes an API call, as this will make the test impure (a test that depends on an external factor is not reliable)_
+
+6) Host your project on Heroku, see [resources](https://github.com/foundersandcoders/master-reference/blob/master/coursebook/week-5/resources.md)
+
+7) Use module.exports and require to break a single large server file into smaller modules.
+
+8) Consider a good server file structure based on what we have discussed over the week.
+
+9) Employ continuous integration on your project with Travis or a similar tool. (If you decide to use Travis, we strongly recommend that you host this project in your own repo rather than in your cohort's FAC repository to avoid all builds getting queued together)
```

Comment:
> (If you decide to use Travis, we strongly recommend that you host this project in your own repo rather than in your cohort's FAC repository to avoid all builds getting queued together)

I think it's fine in our organization